sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/command/AlterTableAddPartitionSuite.scala

```diff
@@ -24,5 +24,21 @@ import org.apache.spark.sql.execution.command.v1
  * V1 Hive external table catalog.
  */
 class AlterTableAddPartitionSuite
-    extends v1.AlterTableAddPartitionSuiteBase
-    with CommandSuiteBase
+  extends v1.AlterTableAddPartitionSuiteBase
+  with CommandSuiteBase {
+
+  test("hive client calls") {
+    withNamespaceAndTable("ns", "tbl") { t =>
+      sql(s"CREATE TABLE $t (id int, part int) $defaultUsing PARTITIONED BY (part)")
+      sql(s"INSERT INTO $t PARTITION (part=0) SELECT 0")
+
+      checkHiveClientCalls(expected = 14) {
+        sql(s"ALTER TABLE $t ADD PARTITION (part=1)")
+      }
+      sql(s"CACHE TABLE $t")
+      checkHiveClientCalls(expected = 14) {
```

Comment:
The number is the same because `ALTER TABLE .. ADD PARTITION` doesn't refresh the cache. I am going to address this issue soon.